app/services/youtube_service.rb

```diff
@@ -1,10 +1,26 @@
 class YoutubeService
   def video_info(id)
     params = { part: 'snippet,contentDetails,statistics', id: id }
-
     get_json('youtube/v3/videos', params)
   end
 
+  def playlist_info(id)
+    params = { part: 'snippet,contentDetails', playlistId: id, maxResults: 50}
+ 
+     raw_videos= get_json('youtube/v3/playlistItems', params)
+       next_page = raw_videos[:nextPageToken]
+    while next_page
+      params = { part: 'snippet, contentDetails', playlistId: id, maxResults: 50, pageToken: next_page }
```

Comment:
We were getting an error because this line was missing contentDetails. Adding it in fixed the problem!

Reply:
Cool! I'm glad it's fixed.